packages/cerebral/src/Controller.js

```diff
@@ -130,6 +130,34 @@ class Controller extends EventEmitter {
     }
 
     return this.runSignal.bind(this, path, signal)
+  } 
+
+  getSignals() {
+    const controller = this
+    function extractSig(path, signals) {
+      return Object.keys(signals).reduce((sigs, key) => {
+        sigs[key] = controller.runSignal.bind(controller, `${path}.${key}`, signals[key])
+        return sigs
+      }, {});
+    }    
+    function getModuleSignals(path, module) {
```

Comment:
Hm, now this functions recreated each getSignals call. But it used only once.
And i just remembered that we wanted to allow delayed module registration.
Lets don't hurry now and wait @christianalfoni answer